.github/workflows/crowdin-download.yml

```diff
@@ -0,0 +1,102 @@
+name: Crowdin Download
+on:
+  workflow_dispatch:
+  # schedule:
+  #   # runs every day at 12:00 noon UTC
+  #   - cron: '0 12 * * *'
```

Comment:
A note that we'll want to uncomment this when we're ready to start downloading the first language.